js/ui.js

```diff
@@ -0,0 +1,112 @@
+$(document).ready(function() {
+	var token = getToken();
+	if (token) {
+		act.update(token).then(replaceUI);
+	}
+})
+
+function login(pass) {
+	act.login(pass).then(function() {
+		saveToken();
+		replaceUI();
+	});
+}
+
+function saveToken() {
+	// saving the token as a cookie, rather than local storage, under the assumption that people are in the habit of clearing cookies specifically when they want to de-auth a site
+	document.cookie = 'chat_token=' + act.token;
```

Comment:
this is fine, just adding a comment to call out the security concern of browser extensions and compromised cookies. (this will always be the case with client side code)